src/net/sourceforge/kolmafia/request/GenericRequest.java

```diff
@@ -1394,11 +1394,11 @@ public void externalExecute() {
   public static final boolean shouldIgnore(final GenericRequest request) {
     String requestURL = GenericRequest.decodeField(request.formURLString);
     return requestURL == null
-        ||
+        //        ||
```

Comment:
Absolutely not. You may not submit these private changes.

Reply:
Fixed.